src/main/scala/no/ndla/articleapi/ArticleApiProperties.scala

```diff
@@ -35,7 +35,7 @@ object ArticleApiProperties extends LazyLogging {
   val MetaInitialConnections = 3
   val MetaMaxConnections = 20
 
-  val AttachmentStorageName = s"$Environment.attachments.ndla"
+  val AttachmentStorageName = if (Environment == "local") s"test.article-attachments.ndla" else s"$Environment.article-attachments.ndla"
```

Comment:
Det føltes litt unødvendig å lage en egen bucket for local import. I og med at vi har test som er "vårt" syntes jeg den passet til lokale imports også